extensions/amp-story/1.0/test/test-amp-story-page.js

```diff
@@ -140,6 +140,13 @@ describes.realWin('amp-story-page', {amp: {extensions}}, (env) => {
     expect(spy).to.have.been.calledOnce;
   });
 
+  it('should call beforeVisible after layoutCallback resolves', async () => {
```

Comment:
did you mean to call `beforeVisible` in this test?

Reply:
I am still confused about this one as well. What exactly are we trying to test here? Now you are stubbing `beforeVisible` but not calling it. If the intention is to ensure `renderOpenAttachmentUI_` is called in `beforeVisible` it might look something like 
```js
    const spy = env.sandbox.spy(page, 'renderOpenAttachmentUI_');
    page.buildCallback();
    await page.layoutCallback();
    page.beforeVisible();
    expect(spy).to.have.been.calledTwice;
```
even better if you can write the test without stubbing private methods if possible e.g.
```js
    page.buildCallback();
    await page.layoutCallback();
    // pseudocode
    expect(querySelector('ctaOutlink')).not.to.exist
    appendChild('amp-story-page-outlink');
    page.beforeVisible();
    expect(querySelector('ctaOutlink')).to.exist 
```